integration/k8s-client.go

```diff
@@ -121,6 +130,7 @@ func WaitForDeployment(kubectlBinary string, kubectlOpts *commands.KubectlOption
 		case <-ticker.C:
 			output, err := commands.RunKubectlRolloutDeployment(kubectlBinary, kubectlOpts, revision)
 			if err != nil {
+				retry++
```

Comment:
nit: the condition below `retry%10 == 0` is always `true` because we never increment the counter